pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -1086,13 +1086,77 @@ func validateValidatingAdmissionPolicyBindingSpec(spec *admissionregistration.Va
 			allErrors = append(allErrors, field.Invalid(fldPath.Child("policyName"), spec.PolicyName, msg))
 		}
 	}
-	allErrors = append(allErrors, validateParamRef(spec.ParamRef, fldPath.Child("paramRef"))...)
+	allErrors = append(allErrors, validateBindingParamRef(spec.ParamRef, fldPath.Child("paramRef"))...)
 	allErrors = append(allErrors, validateMatchResources(spec.MatchResources, fldPath.Child("matchResouces"))...)
 	allErrors = append(allErrors, validateValidationActions(spec.ValidationActions, fldPath.Child("validationActions"))...)
 
 	return allErrors
 }
 
+func validateBindingParamRef(pr *admissionregistration.BindingParamRef, fldPath *field.Path) field.ErrorList {
+	var allErrors field.ErrorList
+	if pr == nil {
+		return allErrors
+	}
+
+	switch pr.Kind {
+	case "":
+		allErrors = append(allErrors, field.Required(fldPath.Child("kind"), ""))
+	case admissionregistration.BindingParamKindGlobal:
+		if pr.GlobalParamRef == nil {
+			allErrors = append(allErrors, field.Required(fldPath.Child("globalParamRef"), fmt.Sprintf("required if Kind is `%s`", admissionregistration.BindingParamKindGlobal)))
+		} else {
+			allErrors = append(allErrors, validateParamRef(pr.GlobalParamRef, fldPath.Child("globalParamRef"))...)
+		}
+	case admissionregistration.BindingParamKindNamespace:
```

Comment:
need to ensure that `GlobalParamRef` is nil

Reply:
added checking for this condition